utils/python/tests/scripts_regression_tests

```diff
@@ -134,7 +134,9 @@ class B_TestCreateNewcase(unittest.TestCase):
 ###############################################################################
     def setUp(self):
         self._testroot = MACHINE.get_value("CESMSCRATCHROOT")
-        self._testdir = os.path.join(self._testroot, tempfile.mktemp())
```

Comment:
I couldnt get mktemp to work correctly on blues so I just picked a case name and hardcoded it.